libs/test/SmartStoreTest/src/com/salesforce/androidsdk/store/SmartStoreTestCase.java

```diff
@@ -57,24 +57,25 @@
 	protected Context targetContext;
 	protected SQLiteOpenHelper dbOpenHelper;
 	protected SmartStore store;
+	protected DBHelper dbHelper;
 
 	@Override
 	public void setUp() throws Exception {
 		super.setUp();
 		EventBuilderHelper.enableDisable(false);
 		targetContext = getInstrumentation().getTargetContext();
 		dbOpenHelper = DBOpenHelper.getOpenHelper(targetContext, null);
-		DBHelper.getInstance(dbOpenHelper.getWritableDatabase(getPasscode())).reset(targetContext, null);
-		store = new SmartStore(dbOpenHelper, getPasscode());
-		store.dropAllSoups();
+		dbHelper = DBHelper.getInstance(dbOpenHelper.getWritableDatabase(getEncryptionKey()));
```

Comment:
We are doing the cleanup in tearDown so as to leave a clean slate to the next tests.